VideoApi/VideoApi.UnitTests/Controllers/ConferenceManagement/StartVideoHearingTests.cs

```diff
@@ -35,7 +35,8 @@ public async Task should_return_accepted_when_start_hearing_has_been_requested()
                 Layout = layout,
                 TriggeredByHostId = hostId,
                 MuteGuests = true,
-                Hosts = hostIds
+                Hosts = hostIds,
+                HostsForScreening = new List<Guid>(),
```

Comment:
could we set this to a non-empty list? Then we can verify below that they are passed to the video platform service

```
VideoPlatformServiceMock.Verify(
                x => x.StartHearingAsync(conferenceId, request.TriggeredByHostId.ToString(), participantIdsAsStrings,
                    hostIdsAsStrings, Layout.ONE_PLUS_SEVEN, muteGuests, hostsForScreening), Times.Once);
```

Reply:
done